example_app_frames/src/main/java/com/checkout/example/frames/ui/utils/CustomPaymentFormTheme.kt

```diff
@@ -86,12 +86,19 @@ object CustomPaymentFormTheme {
         .setSubTitleTextId(R.string.cko_cvv_component_subtitle)
         .build()
 
-    private val billingSummary = PaymentFormComponentBuilder()
-        .setPaymentFormField(PaymentFormComponentField.BillingSummary)
+    private val billingSummaryTextStyle = PaymentFormComponentBuilder()
+        .setPaymentFormField(PaymentFormComponentField.BillingSummaryTextStyle)
         .setTitleTextId(R.string.cko_billing_address)
+        .setIsFieldOptional(false)
         .setSubTitleTextId(R.string.cko_billing_address_info)
         .build()
 
+    private val billingSummaryContainer = PaymentFormComponentBuilder()
```

Comment:
It looks a bit confusing to me because of `container` word in `BillingSummaryContainer`.
Could it be BillingSummaryButton?

And instead of `BillingSummaryTextStyle` could we use `BillingSummaryPreview`?

Reply:
@maxim-nosov-cko BillingSummaryContainer stands for AddressSummaryComponentStyle, it does not cover only the button but edit billing address style as well. Can we rename it as addressSummaryComponent. Additionally, BillingSummaryTextStyle is for styling part for title, subtitle of this component. It does not stands for edit address text.